droid-hal-device.inc

```diff
@@ -373,10 +373,10 @@ fi
 %if 0%{?_obs_build_project:1}
 
 # Set up kernel extra version for OBS kernel builds
-if EXTRAVERSION_OLD=$(cat %android_root/kernel/Makefile | grep "EXTRAVERSION ="); then
-    sed -i "s/$EXTRAVERSION_OLD/EXTRAVERSION = +%{version}/g" %android_root/kernel/Makefile
-elif EXTRAVERSION_OLD=$(cat %android_root/linux/kernel/Makefile | grep "EXTRAVERSION ="); then
-    sed -i "s/$EXTRAVERSION_OLD/EXTRAVERSION = +%{version}/g" %android_root/linux/kernel/Makefile
+if extra_version_old=$(grep"EXTRAVERSION =" %android_root/kernel/Makefile); then
```

Comment:
```extra_version_old``` - no such variable referenced afterwards
```grep"EXTRAVERSION ="``` - space missing (will fail as command)
Have you tested this .spec in action?

Reply:
Yes I had after I fixed the typo.